Lib/test/test_complex.py

```diff
@@ -364,6 +364,11 @@ def test_pow(self):
                     except OverflowError:
                         pass
 
+        # Check that complex numbers with special components
+        # are correctly handled.
+        self.assertComplexesAreIdentical(complex(1, +0.0)**2, complex(1, +0.0))
+        self.assertComplexesAreIdentical(complex(1, -0.0)**2, complex(1, -0.0))
```

Comment:
You could actually have more test cases if you want?

Reply:
Yes, that's why I also referenced #60200.  We can include more test cases in some new file of Lib/test/mathdata/.   But I'm not sure this belongs to the current pr.

Reply:
I like having extensible test coverage so I would be in favor of adding more cases in this PR since we are removing a functionality. But let's leave this decision to the core dev merging it.